internal/config/cors.go

```diff
@@ -13,19 +13,26 @@ type CorsConfig struct {
 	Enabled        bool     `json:"enabled" mapstructure:"enabled" yaml:"enabled"`
 	AllowedOrigins []string `json:"allowedOrigins,omitempty" mapstructure:"allowed_origins" yaml:"allowed_origins,omitempty"`
 	AllowedHeaders []string `json:"allowedHeaders,omitempty" mapstructure:"allowed_headers" yaml:"allowed_headers,omitempty"`
+	AllowedMethods []string `json:"allowedMethods,omitempty" mapstructure:"allowed_methods" yaml:"allowed_methods,omitempty"`
 }
 
 func (c *CorsConfig) setDefaults(v *viper.Viper) error {
 	v.SetDefault("cors.enabled", false)
 	v.SetDefault("cors.allowed_origins", "*")
 	v.SetDefault("cors.allowed_headers", []string{
+		"Origin",
```

Comment:
```suggestion
		"Origin",
		"X-Flipt-Accept-Server-Version",

Reply:
this header is missing

Reply:
we dont use that in v2, i should remove it from the schemas